docs/guides/cloud/account-management/projects.mdx

```diff
@@ -299,6 +299,35 @@ See also
 [Highlight your project's test status with Cypress README badges](https://www.cypress.io/blog/2020/09/02/highlight-your-projects-test-status-with-cypress-readme-badges/)
 announcement.
 
+## Best Practices
+
+### Multi-repo Implementation
+
+There is a sceario where you may have Cypress tests in a repo separate from the application code itself. This creates a sub-optimal [Branch Review](guides/cloud/branch-review) experience considering the commit and branch interpretation of each git directory.
```

Comment:
```suggestion
There is a scenario where you may have Cypress tests in a repo separate from the application code itself. This creates a sub-optimal [Branch Review](guides/cloud/branch-review) experience considering the commit and branch interpretation of each git directory.
```

...though I wonder if we need to explicitly call out branch review here; it'd be problematic for really any sort of VCS integration we have, including statuses/PR comments, etc.

Reply:
Makes sense. I figured it may be useful to callout a practical area in the product this would impact but we can make it more general to VC. 

@pstakoun any other thoughts so I can batch changes?

Reply:
I agree with making it more general and also mention Branch Review as an example of a feature that is impacted.

Reply:
@pstakoun  I took out the direct mention of branch review in the last commit. Are you saying we should still leverage it in some way?

Reply:
nah this is fine. I'd just maybe link to this section from within the branch review docs